data/ngoApplyData.js

```diff
@@ -115,7 +126,7 @@ export default {
     {
       question: "Is there a development fee?",
       answer:
-        "We typically charge a fee of $1,000 to our clients for any work that we do. However, this fee can be reduced on a need basis."
+        "We typically charge a fee of $1,000 per semester to our clients for any work that we do. As a student-led nonprofit building products for other nonprofits, we typically incur costs from using services like Amazon AWS or other third-party APIs as we build our products. We also aim to develop our members professionally and foster a collaborative, passionate, and dedicated community by offering tech talks, networking events, and organization-wide retreats. Thus, the development fee mainly goes towards technology costs, professional development, and community events. However, we understand that nonprofits may have different financial circumstances and would be more than happy to reduce the fee or make accommodations on a need basis."
```

Comment:
Lets not say "happy to reduce the fee", but be a little more stern here (and honest).

Reply:
not happy to reduce the fee